app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee

```diff
@@ -17,6 +18,13 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
       @initializeTree()
       @filterSwitchView.filterInit()
 
+  removeLinkedDocument: (e) ->
+    e.preventDefault()
+    item = e.currentTarget
+    container = @$(item).closest('.linked-document')
+    @$(container).find('input[type="text"]').val('')
```

Comment:
`container` should already be a jQuery object, so you shouldn't need the `@$()` here.